src/electron-main.js

```diff
@@ -386,50 +386,18 @@ ipcMain.on('ipcCall', async function(ev, payload) {
                 mainWindow.webContents.goForward();
             }
             break;
-        case 'setLanguage': {
-            // work around `setSpellCheckerLanguages` being case-sensitive by converting to expected case
-            const caseMap = {};
-            const availableLanguages = mainWindow.webContents.session.availableSpellCheckerLanguages;
-            availableLanguages.forEach(lang => {
-                caseMap[lang.toLowerCase()] = lang;
-            });
-
-            if (!caseMap["en"]) {
-                // default special-case for `en` as in Riot is actually implies `en-GB`. `en-US` is distinct.
-                // this way if `en` is requested and not available and `en-GB` is available it'll be used.
-                caseMap["en"] = caseMap["en-gb"];
+        case 'setSpellCheckLanguages':
+            if (args[0] && args[0] != []) {
+                mainWindow.webContents.session.setSpellCheckerLanguages(args[0]);
```

Comment:
This function will throw an error if any of the language code are unexpected, e.g. perhaps they change over time. Are we handling that properly?

Reply:
On the React side, we are retrieving the languages directly from Electron. So I think this can't happen. But I can add a try-catch block to make this nicer